contracts/poll/BasePoll.sol

```diff
@@ -122,6 +122,7 @@ contract BasePoll {
         return true;
     }
 
+    // why are there 2 view methods with the same return value?
```

Comment:
@Evert0x The result of removing logic and not refactoring the result:) Agreed on simplifying this.